src/openrct2/Context.cpp

```diff
@@ -947,16 +948,23 @@ namespace OpenRCT2
             RunGameLoop();
         }
 
+        bool ShouldDraw()
+        {
+            if (gOpenRCT2Headless)
+                return false;
+            if (_uiContext->IsMinimised())
+                return false;
```

Comment:
```suggestion
            if (gOpenRCT2Headless || _uiContext->IsMinimised())
                return false;
```

Reply:
Running headless and minimized are not related, so I don't see the added value of combining the two on one line.